staking-cli/src/registration.rs

```diff
@@ -210,6 +243,44 @@ mod test {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_update_commission() -> Result<()> {
```

Comment:
I think we should also test that a commission update fails if it doesn’t respect the update interval

Reply:
This is tested in the solidity tests but I'll add it here as well.